indi-svbony/svbony_ccd.h

```diff
@@ -109,6 +109,11 @@ class SVBONYCCD : public INDI::CCD
         // pixel size
         float pixelSize;
 
+        // Camera Firmware version number
+        char cameraFirmwareVersion[65];
+        // SVBONY Camera SDK version number
+        const char* SDKVersion;
```

Comment:
If you're only using this to report version information once in Connect, is there a reason to keep it as class variable? or is this used elsewhere?

Reply:
If EKOS is used, this information is output to the log and need not be retained.
If non-EKOS clients use it without logging, I will retain this information for reference when debugging.
We also plan to display this information in the INDI control panel in the future, so we decided to keep it.